src/puppetlabs/jdbc_util/core.clj

```diff
@@ -156,3 +156,42 @@
                        [sql 0]
                        seq-params-w-indices)]
     (vec (conj (flatten parameters) sql'))))
+
+(defn has-extension? [db extension]
+  (-> (jdbc/query db ["select count(*) from pg_extension where extname = ?" extension])
+      first
+      :count
+      pos?))
+
+(defn has-pglogical-extension? [db]
+  (has-extension? db "pglogical"))
+
+(defn- unsafe-escape-sql-quotes
+  "Escape the given string so it can be \"safely\" passed as a string parameter
+  to an sql query."
+  [s]
+  (clojure.string/replace s "'" "''"))
+
+(defn wrap-ddl-for-pglogical
+  "Wrap the given sql (presumably DDL) in a call to
+  pglogical.replicate_ddl_command, escaping quotes and wrapping the statement so
+  it won't return anything."
+  [sql schema]
+  (str "do 'begin perform "
+       (unsafe-escape-sql-quotes
+        (str "pglogical.replicate_ddl_command('"
+             "set local search_path to " schema "; "
```

Comment:
if you were going to quote the schema as a identifier this seems like it would be the place